src/Interfaces/VTK.jl

```diff
@@ -23,7 +23,7 @@ function write_vtk(sol::AbstractSolution, filename::String)
     msh = mesh(structure(analysis(sol)))
     nodes_mat = node_matrix(msh)
     connec_mat = connectivity(msh)
-    num_elem = length(elements(msh))
+    num_elem = num_elements(msh)
     cells = [MeshCell(VTKCellTypes.VTK_TETRA, connec_mat[e]) for e in 1:num_elem]
```

Comment:
I think here we need to define a set of `cell_to_vtkcell(::Type{ONSAS.Elements.Tetrahedron}) = VTKCellTypes.VTK_TETRA` and constructors for `VTKFile` directly. WDYT ?

Reply:
Sounds good 👍